panel/auth.py

```diff
@@ -54,6 +54,7 @@ def decode_response_body(response):
         body = codecs.decode(response.body, 'ascii')
     except Exception:
         body = codecs.decode(response.body, 'utf-8')
+    body = re.sub("\'", '\\"', body)
```

Comment:
This is a workaround as just below you replace both " and '. It works. But its hard to do right as there are no existing test cases.